reana_commons/config.py

```diff
@@ -420,6 +420,9 @@ def default_workspace():
 )
 """Kubernetes valid memory format regular expression e.g. Ki, M, Gi, G, etc."""
 
+KUBERNETES_CPU_FORMAT = r"^(?P<value_millicpu>[1-9]\d*)m$|^(?P<value_cpu>(0*[1-9]\d*(\.\d+)?|0*\.\d*[1-9]\d*))$"
+"""Kubernetes valid CPU format regex. Supports formats like 500m, 1, 0.1. Values must be greater than 0."""
```

Comment:
Cosmetics: please put the example values in ascending order, so... "0.1" (or "100m"), "0.9" (or "900m"). This would also gently hint that 0.1=100m and that the two value systems are just two ways how to express the same thing.

Example: Valid K8s CPU core limit format, such as "0.1" (or "100m"), "0.9" (or "900m").